src/css/style.css

```diff
@@ -67,7 +120,49 @@ a:hover {
   .navMenuButton {
     @apply hidden;
   }
+
+  footer {
+    height: 32em;
+  }  
+
+  .socials {
+    font-size: 80px;
+  }
+
+  .socials a {
+    @apply mx-6
+  }
+
+  .socials, .emailAndCopyright {
+    height: 50%;
+  }
+
+  .emailAndCopyright {
+    @apply flex flex-row justify-around items-center
+  }
+
+  .emailLink {
+    @apply justify-start
+  }
+
+  .emailName {
+    @apply pl-4
+  }
+  
+
+  .connectLogo {
+    transform: translateY(-6.35em);
+    margin-left: 4em;
+    width: 27em;
+  }
+  
   body {
     @apply bg-FCCblue-100;
   }
+}
+
+@screen xl {
+  .socials a {
+    @apply mx-16
+  }
```

Comment:
Is this something that a designer asked you to add? I think it looks good, I just wasn't sure if it was intentional.

Reply:
Its been a minute since I've looked at this, but IIRC when the window was about tablet sized, the links/icons were just a bit too big, really just nit picky stuff. To answer your question, no a designer didn't ask me to add this, just added it myself as it seemed like the right move.

Reply:
`@screen xl` applies at resolutions larger than tablet, if I recall correctly. More importantly, I don't want too many breakpoints so that elements are jumping around as the windows resize. Just once for mobile, once for desktop, and possibly one more for tablet (something that Ellen's looking at right now). If it's not too much trouble, I'd appreciate removing this for now and we can reexamine later if it'd help. Thank you!